functions/src/definitions/eventHandlers/onInstanceUpdate.ts

```diff
@@ -4,30 +4,35 @@ import {
   CollectionTypes,
 } from "../common/typesense/collectionOperations"
 import { getEmbedding } from "../common/machineLearningServer/operations"
+import { onDocumentUpdated } from "firebase-functions/v2/firestore"
 
-const onInstanceUpdate = functions
-  .region("asia-southeast1")
-  .runWith({
+const onInstanceUpdateV2 = onDocumentUpdated(
+  {
+    document: "/messages/{messageId}/instances/{instanceId}",
```

Comment:
```suggestion
    document: "messages/{messageId}/instances/{instanceId}",
```

To standardise with no leading slash

Reply:
Done